packages/playground/app/docs/getting-started.html

```diff
@@ -0,0 +1,100 @@
+---
+layout: default
+title: Getting Started
+nav_order: 1
+permalink: /playground
+---
+
+<style>
+	@font-face {
+		font-family: "72-Bold";
+		src: url(../assets/fonts/72-Bold.woff2) format("woff2"),
+				url(../assets/fonts/72-Bold.woff) format("woff");
+	}
+
+	@font-face {
+		font-family: "72-Regular";
+		src: url(../assets/fonts/72-Regular.woff2) format("woff2"),
+				url(../assets/fonts/72-Regular.woff) format("woff");
+	}
+
+	body {
+		font-family: '72-Regular';
+	}
+
+	.wrapper {
+		width: 100%;
+		max-width: 1000px;
+		margin-top: 3rem;
+		box-sizing: border-box;
+	}
+
+	.wrapper > h1 {
+		margin: 3rem 0;
+		font-size: 1.7rem;
+		font-family: "72-Bold";
+	}
+
+	.setup-wrapper {
+		margin-top: 4rem;
+	}
+
+	.setup-wrapper > h2 {
+		font-family: "72-Bold";
+	}
+
+	.step {
+		margin-bottom: 2rem;
+	}
+
+	.align-vertically {
+		display: flex;
+		align-items: center;
+	}
+
+	pre.highlight {
+		padding-bottom: 0;
+	}
+
+	@media screen and (max-width: 600px) {
+		.wrapper {
+			padding: 0 1.5rem;
+		}
+	}
+</style>
+<section class="wrapper">
```

Comment:
Can we exclude the content of this page as an .md file so I can be easily modified by the doc guys if needed?
It looks like a pretty static content

Just a suggestion, not sure if we need it

Reply:
Done